x-pack/filebeat/module/cisco/amp/_meta/fields.yml

```diff
@@ -0,0 +1,276 @@
+- name: cisco.amp
+  type: group
+  release: beta
+  default_field: false
+  description: >
+    Module for parsing Cisco AMP logs.
+  fields:
+    - name: timestamp_nanoseconds
+      type: date
+      description: >
+        The timestamp in Epoch nanoseconds.
+
+    - name: date
+      type: date
+      description: >
+        The timestamp in ISO8601 format.
```

Comment:
Will have a specific fixed timezone? If so maybe is good to document it.

Reply:
This field has since been removed, they have 3 different fields for the same timestamp. These are since moved to ECS fields.